src/app/src/main/kotlin/org/vaccineimpact/orderlyweb/controllers/web/ReportController.kt

```diff
@@ -48,16 +48,20 @@ class ReportController(
     @Template("run-report-page.ftl")
     fun getRunReport(): RunReportViewModel
     {
+        val reportKey = "report-name"
+        val reportName = context.queryParams(reportKey).toString()
+
+        val queryParams: Map<String, String> = mapOf(reportKey to reportName)
         val metadata = orderlyServerAPI
                 .throwOnError()
-                .get("/run-metadata", context)
+                .get("/run-metadata", context, queryParams.filter { it.key != reportKey })
```

Comment:
This will accept other queryParams except report-name. If this is set to emptyMap() then we will be revisiting this code again when the need to send a different queryParam for this endpoint. Let me know if I am missing something.